File: 240102/두 배열의 곱/multiple-of-two-arrays.py
arr = []
arr2 = []

for _ in range(3):
    arr.append(list(map(int,input().split())))
input()
for _ in range(3):
    arr2.append(list(map(int,input().split())))

tot_arr = [[0]*3 for _ in range(3)]

for i in range(3):
    for j in range(3):
        tot_arr[i][j] = arr[i][j] * arr2[i][j]
        print(tot_arr[i][j],end=" ")
    print()

File: 240102/두 배열의 곱/multiple-of-two-arrays.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int[][] arr = new int[3][3];
        int[][] arr2 = new int[3][3];
        int[][] total = new int[3][3];
        for(int i = 0 ; i<3; i++){
            for(int j = 0 ; j<3; j++){
                arr[i][j] = sc.nextInt();
            }
        }
        
        for(int i = 0 ; i<3; i++){
            for(int j = 0 ; j<3; j++){
                arr2[i][j] = sc.nextInt();
            }
        }
        for(int i = 0 ; i<3; i++){
            for(int j = 0 ; j<3; j++){
                total[i][j] = arr[i][j] * arr2[i][j];
                System.out.print(total[i][j]+" ");
            }
            System.out.println();
        }
    }
}